packages/php-wasm/universal/src/lib/universal-php.ts

```diff
@@ -6,6 +6,7 @@ import { PHPResponse } from './php-response';
  */
 export interface PHPRequestEndEvent {
 	type: 'request.end';
+	data?: {[log: string]: string};
```

Comment:
@adamziel do you see any issues with using the `request.end` event to send logs back to the browser?

Reply:
True, I had the impression that we wouldn't be able to make PHP requests after an error, but it seems to work fine if Logger requests the log file.